paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutInteractor.kt

```diff
@@ -28,6 +28,7 @@ internal interface PaymentMethodVerticalLayoutInteractor {
     sealed interface ViewAction {
         data object TransitionToManageSavedPaymentMethods : ViewAction
         data class PaymentMethodSelected(val selectedPaymentMethodCode: String) : ViewAction
+        data class SavedPaymentMethodSelected(val savedPaymentMethod: DisplayableSavedPaymentMethod) : ViewAction
```

Comment:
Should we pass in just the `PaymentMethod`, since I think that's all that's used.

Reply:
sounds good! Updated